src/pmi/bgq/bgq_pmi.c

```diff
@@ -0,0 +1,66 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2001 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+
+/*********************** PMI implementation ********************************/
+/*
+ * This file implements the client-side of the PMI interface.
+ *
+ * Note that the PMI client code must not print error messages (except
+ * when an abort is required) because MPI error handling is based on
+ * reporting error codes to which messages are attached.
+ *
+ * In v2, we should require a PMI client interface to use MPI error codes
+ * to provide better integration with MPICH.
+ */
+/***************************************************************************/
+
+#include "mpichconf.h"
+
+#include <stdio.h>
```

Comment:
I think all this boiler plate can be removed, right? We can just `#include "pmi.h"` and that should be enough.